Exercise_D and E/assets/style/index.css

```diff
@@ -2,3 +2,203 @@
 This css sheet is linked to practice.html
 You can style tags, classes (.) or ids (#) from this file.
 */
+html {
+    font-family: Arial, Helvetica, sans-serif;
+}
+
+@media (max-width: 480px) {
+    #call-to-action {
+        display: flex;
+        flex-direction: column;
+        align-items: center;
+        position: absolute;
+        top: 35%;
+        left: 6%;
+        z-index: 1;
+        min-width: 90vw;
+    }
+
+
```

Comment:
Stick to one colour convention within your css sheet, this is a convention that companies follow, they would usually tell you what colours to use, and what format they should be in.